ext/easylogging-8.91/easylogging++.h

```diff
@@ -2249,6 +2255,16 @@ class Logger {
         return configured_;
     }
 
+    // no checks performed
+    // target must exist longer, than logger OR be removed
+    inline void addLogTarget(ILogTarget* target) {
+        customTargets_.push_back(target);
+    }
+
+    inline void removeLogTarget(ILogTarget* target) {
+        customTargets_.remove_if(ShouldDelete(target));
```

Comment:
Not sure, is it any differences with customTargets_.remove(target);
Or you plan extend ShouldDelete in future?